docs/source/algorithms/GeneratePeaks-v1.rst

```diff
@@ -116,7 +116,7 @@ Generate peaks from a TableWorkspace
 .. testcleanup:: GeneratePeakFromTable
 
   DeleteWorkspace(Workspace=outws)
-  DeleteWorkspace(Workspace="focussed")
+  DeleteWorkspace(Workspace="focused")
```

Comment:
This change is causing a doc test to fail. You don't need to *fix* any of the **focussed** words as this is correct for British English.

Reply:
Reverted in 7b9f043